frontstage/views/sign_in/sign_in.py

```diff
@@ -37,22 +45,18 @@ def login():
 
         party_json = party_controller.get_respondent_by_email(username)
         if not party_json or 'id' not in party_json:
-            logger.info('Respondent not able to sign in as they don\'t have an active account in the system.')
+            logger.info('Respondent not able to sign in as they don\'t have an account in the system.')
             return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
         party_id = party_json['id']
 
+        # Take our raw token and add a UTC timestamp to the expires_at attribute
         try:
             oauth2_token = oauth_controller.sign_in(username, password)
         except OAuth2Error as exc:
-            error_message = exc.oauth2_error
-            if BAD_AUTH_ERROR in error_message:
-                return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
-            elif NOT_VERIFIED_ERROR in error_message:
-                logger.info('User account is not verified on the OAuth2 server')
-                return render_template('sign-in/sign-in.account-not-verified.html', form=form)
-            else:
-                logger.info('OAuth 2 server generated 401 which is not understood', oauth2_error=error_message)
-                return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
+            route_validator = _ERROR_ROUTES.get(exc.oauth2_error,
+                                                RenderUnVerifiedError('sign-in/sign-in.html', _DATA))
+            return route_validator.log_message(exc.oauth2_error).\
+                notify_user(username, party_json['lastName'] + ' ' + party_json['firstName']).route_me(form)
```

Comment:
Has this format of `lastName firstName` been agreed with anyone (BAs/PO)? It doesn't seem right and other email notifications just use first name (e.g. https://github.com/ONSdigital/ras-party/blob/dc56d4aa426a0e4b6d275030b6b5d5ef9ca8d794/ras_party/controllers/account_controller.py#L268)

Reply:
Maybe the card should have been more specific.  You could use anything as a name if you wanted to but I'd say that we should be consistent across all interaction with respondents.  Cheers for checking and changing it :-)